scopes/component/component-descriptor/aspect-map.ts

```diff
@@ -0,0 +1,33 @@
+export type AspectDataEntry = {
+  aspectId: string;
+  aspectData: string;
+};
+
+export type AspectMapProps = {
+  entries?: AspectDataEntry[];
+};
+
+export class AspectMap {
```

Comment:
we have already a very similar data structure, we shouldn't create a new one
see here - https://github.com/teambit/bit/blob/master/scopes/component/component/aspect-list.ts

Reply:
i'm using it also on my side so I don't want to change the name